### ISSUE TITLE
[Feature Request] Togglable features

### ISSUE THREAD

**Issue by DutchPete** [mulfok/obsidian-absolve#2]
Users might not want to use all of Absolve's features, so it would be useful to be able to have enable/disable toggles.

For example: I prefer to have my own headings snippet prevail, but when I use Absolve that snippet is overridden by Absolve. So, what I did as a workaround is commented out heading code in Absolve that is overriding my snippet.

Obviously, that is not the way to use Absolve. Like with someone's theme I use, I don't want to have to interfere in the original CSS.

**Comment by mulfok** [comment]
This is something that I had been silently working on—the new release that (should) come up in just a couple of minutes fixes this. Have a little evaluation of it, and you feel that it meets everything, feel free to close this issue

**Comment by DutchPete** [comment]
I downloaded `absolve.css` from the green Code button, but notice that the version is still at 0.1.0.

When I downloaded it yesterday 30 mins. after you posted your comment, it was also at 0.1.0. If I downloaded from the wrong place, please point me in the right direction.

**Comment by mulfok** [comment]
no worries, you had actually downloaded it fine, I had just forgotten to update the version number at the top. oops! 😵 
I've pushed a new release anyways (https://github.com/mulfok/obsidian-absolve/releases/tag/v2.1.0-beta) in the future, you should download from the releases tab, as I sometimes push unfinished stuff to the source code. Let me know how it works!

**Comment by DutchPete** [comment]
Thanks. I just downloaded it, and I see toggles under the heading *Tweaks* and under the heading *External theme features*, the rest of features do not have toggles.

Having said that, my headings don't seem to be affected by the snippet, i.e. my own header snippet prevails.

In the previous Absolve version I had set the font family to what I wanted, and that was taken on by the latest version without me having had to do anything. That also applies to the tweak *Compact Admonitions*: I toggled it to enable, and that occurred in the new version too. Note, there is a typo: it says Compact Ad**om**nitions.

I had also changed the workspace color scheme to “notepad”, yet that was not taken on by the latest version. Huh?

Don't get me wrong: Absolve works fine, I like it.

**Comment by mulfok** [comment]
Curious.
The reasoning for the headers being overridden is that the current CSS does not have any `!important` tags there, but I see that I'll add them back in.
When it comes to the notepad colour scheme not working, please make sure the following:
- [x] You're using light mode
- [x] You're not using Minimal (_notepad is derived from Minimal's gruvbox implementation_)

I've fixed the spelling error, and I'm racking my for a list of features that I've added to Absolve that I could go and add a toggle to. One of these is Highlighter, but there's a strange bug with the class toggle, so until I have a solution, it will have to remain on by default.

In the meantime, if you could make a list of the things that you would _like_ to have a toggle, it would make my job a lot easier. Thanks!

**Comment by DutchPete** [comment]
Re headers overridden: I want to be able to use my own snippet because I prefer that to the Absolve solution. So, please don't use `!important` if it is not necessary.

Re notepad: yes, I am using light mode, and no, I am not using Minimal. The fact that the notepad color was not taken on by the new version is, as far as I am concerned, a minor issue because reselecting it in the dropdown is no big deal.

Re toggles: looking at it in a bit more detail, I have come round to appreciating more what there is, so I can say it is fine as it is.

In conclusion, therefore, I would say: well done so far. I think it is important to keep it easy for users to override certain feature with their own snippet, as per my header issue.

**Comment by mulfok** [comment]
Alrighty! The new update should address some stuff, and add a couple toggles here and there. Since Absolve is in beta, and toggles will be added as I progress, I'm going to close this issue. Thanks for the input however, it's greatly appreciated!

**Comment by DutchPete** [comment]
Just to let you know that after downloading the latest version and replacing the previous version, all my settings were taken on by the new version ! Well done 👍

Also, the icons you added are a nice touch 😀

**Comment by mulfok** [comment]
Glad you like it!
